Source/PLCrashReporterConfig.m

```diff
@@ -134,6 +153,28 @@ - (instancetype) initWithSignalHandlerType: (PLCrashReporterSignalHandlerType) s
                      symbolicationStrategy: (PLCrashReporterSymbolicationStrategy) symbolicationStrategy
     shouldRegisterUncaughtExceptionHandler: (BOOL) shouldRegisterUncaughtExceptionHandler
                                   basePath: (NSString *) basePath
+{
+    return [self initWithSignalHandlerType: signalHandlerType
+                     symbolicationStrategy: symbolicationStrategy
+    shouldRegisterUncaughtExceptionHandler: shouldRegisterUncaughtExceptionHandler
+                                  basePath: basePath
+                            maxReportBytes: MAX_REPORT_BYTES];
+}
+
+/**
+ * Initialize a new PLCrashReporterConfig instance.
+ *
+ * @param signalHandlerType The requested signal handler type.
+ * @param symbolicationStrategy A local symbolication strategy.
+ * @param shouldRegisterUncaughtExceptionHandler Flag indicating if an uncaught exception handler should be set.
+ * @param basePath The base path to save the crash data.
```

Comment:
I would specify that `basePath` can be `nil` to avoid confustion, as now it is required to be specified if you need to set the `maxReportBytes`.